docs/configuration.md

```diff
@@ -477,6 +477,64 @@ Apart from these, the following properties are also available, and may be useful
     <code>spark.io.compression.codec</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.chimera.crypto.cipher.suite</code></td>
+  <td>AES/CTR/NoPadding</td>
+  <td>
+    Cipher suite for crypto codec.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.chimera.crypto.codec.classes</code></td>
+  <td></td>
+  <td>
+    The prefix for a given crypto codec, contains a comma-separated
+    list of implementation classes for a given crypto codec (eg AES/CTR/NoPadding).
+    The first implementation will be used if available, others are fallbacks.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.chimera.crypto.jce.provider</code></td>
+  <td></td>
+  <td>
+    The JCE provider name used in CryptoCodec.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.chimera.crypto.buffer.size</code></td>
+  <td>8192</td>
+  <td>
+  The buffer size used by CryptoInputStream and CryptoOutputStream.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.encrypted.data.keySizeBits</code></td>
+  <td>128</td>
+  <td>
+    Shuffle encrypt data key size default is 128
```

Comment:
"Shuffle file encryption key size." You already said the default is 128 in L512.